medusa/server/web/home/add_shows.py

```diff
@@ -100,7 +100,8 @@ def searchIndexersForShowName(search_term, lang=None, indexer=None):
 
         for i, shows in iteritems(results):
             final_results.extend({(indexerApi(i).name, i, indexerApi(i).config['show_url'], int(show['id']),
-                                   show['seriesname'].encode('utf-8'), show['firstaired']) for show in shows})
+                                   show['seriesname'].encode('utf-8'), show['firstaired'] or 'N/A',
+                                   show['network'] or 'N/A') for show in shows})
```

Comment:
Are there any networks which should be encoded?

Reply:
I don't know. Same applies for show names right? No issues about that